ethereum/handler/ethereumHandler.go

```diff
@@ -161,15 +161,15 @@ func (eh *EthereumHandler) PrintInfo(ctx context.Context, seconds time.Duration)
 	var err error
 	var syncingInfo string
 	for {
+		if ctx.Err() != nil {
+			return
+		}
 		time.Sleep(seconds)
```

Comment:
nit: do a select between ctx.Done and time.After, to stop sleeping if the context is cancelled